cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -548,27 +539,23 @@ def test_guest_os_reset(appliance, testing_vm_tools, ensure_vm_running, soft_ass
     testing_vm_tools.power_control_from_cfme(
         option=testing_vm_tools.GUEST_RESTART, cancel=False, from_details=True)
     view.flash.assert_success_message(text='Restart Guest initiated', partial=True)
-
-    soft_assert(
-        wait_for_last_boot_timestamp_refresh(testing_vm_tools, last_boot_time),
-        "Last Boot Time value has not been refreshed")
+    if not (BZ(1571830, forced_streams=["5.10", "5.11"]).blocks and provider.one_of(RHEVMProvider)):
+        soft_assert(
+            wait_for_last_boot_timestamp_refresh(testing_vm_tools, last_boot_time),
+            "Last Boot Time value has not been refreshed",
+        )
     soft_assert(
         ensure_state_changed_on_unchanged(testing_vm_tools, state_changed_on),
-        "Value of 'State Changed On' has changed after guest restart")
+        "Value of 'State Changed On' has changed after guest restart",
+    )
     soft_assert(testing_vm_tools.mgmt.is_running, "vm not running")
 
 
 @pytest.mark.provider([VMwareProvider, RHEVMProvider], override=True)
-@pytest.mark.meta(
-    blockers=[
-        BZ(
-            1571895,
-            forced_streams=["5.10"],
-            unblock=lambda provider: not provider.one_of(RHEVMProvider),
-        )
-    ]
-)
-def test_guest_os_shutdown(appliance, testing_vm_tools, ensure_vm_running, soft_assert):
+@pytest.mark.meta(blockers=[BZ(1723485, forced_streams=["5.11"],
+                               unblock=lambda provider: not (provider.one_of(RHEVMProvider)
+                                                             and not provider.version < 4.3))])
```

Comment:
Is this test case automating BZ 1723485, if so you can add the automates metadata.

Reply:
Hi @john-dupuy 
- BZ(1571830) is not blocking because of below reasons I found in miq shell. Hence I put forced streams.
```
In [1]: BZ(1571830).blocks
ATTENTION!!: No release flags, wrong versions, ignoring 1571830
Out[1]: False
```
- Yes, this BZ(1723485) automating this test case. So I am updating it for automates meta data.